.github/workflows/maven.yml

```diff
@@ -1,7 +1,7 @@
 # This workflow will build a Java project with Maven
 # For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
 
-name: Kubernetes Java Client - Maven Test
+name: build
```

Comment:
this is for shortening the badge name. currently the build status badge is still referencing travis

![https://github.com/kubernetes-client/java/workflows/Kubernetes%20Java%20Client%20-%20Maven%20Test/badge.svg](https://github.com/kubernetes-client/java/workflows/Kubernetes%20Java%20Client%20-%20Maven%20Test/badge.svg)